src/server.c

```diff
@@ -100,11 +100,26 @@ void get_d20(int fd)
     // IMPLEMENT ME! //
     ///////////////////
 
+    srand(time(NULL));
```

Comment:
if I may suggest: set this inside of `main(){}`, that way you only seed once per server run instead of once per d20 call. you'll still get different random strings

Reply:
I’ve seen some vague allusions to only doing the seed once, but other than possible entropy exhaustion for larger use apps, why do so?

Reply:
Updated as suggested, but would still be interested if you know more about this. :)